docs/en/UI/Angular/Component-Replacement.md

```diff
@@ -77,6 +77,140 @@ export class AppComponent {
 
 ![Layout Components](./images/layout-components.png)
 
+### How to Add a New Layout Component
+
+To add a new layout component, you need to follow these steps:
+
+#### Step 1: Create a New Angular Component
+
+First, you need to create a new Angular component. This component should have a 'router-outlet' for dynamic content loading. You can create a new component using the Angular CLI. Run the following command in your terminal:
```

Comment:
`First, you need to create a new Angular component.` repates title don't need this sentence